options.html

```diff
@@ -96,10 +96,9 @@ <h2>
 
                     <p>
                         <input type="checkbox" id="image_loader_newtab"/>
```

Comment:
I have a suggestion for organizing the embedding options:

---

## Multimedia Embedding
_Show multimedia links inline when clicked._

Embed media types:
- [ ] Images
- [ ] Videos (and □ show in HD when available)
- [ ] Twitter and Instagram posts
- [ ] Shacknews posts

Embedding options:
- [ ] Embed media items near their associated links
- [ ] Open media in a fullscreen overlay when middle-clicked

## Post Enhancements
_Various enhancements to the display of expanded posts._

- [ ] Open NWS links in incognito mode
- [ ] Dino gegtik comic
- [ ] Sparkly comic
- [ ] Work around slow scrolling performance

---

There are enough embed-related options that I think it should have its own top-level section. Stuff like dino gegtik and open NWS links in incognito mode can be separated into their own group. Then I think within the embed section we can make it clear which are 1.) types of media to embed vs. 2.) options for embedding.

Reply:
I agree. In fact, going forward, I was planning a more thorough rework of the overall Options panel as part of the upcoming user-filter enhancements.

I'll see about this particular suggestion sometime this week.